lib/CommandProcessor.js

```diff
@@ -98,7 +98,7 @@ p.process_data = function(chunk) {
 					break;
 				}
 			}
-			while(false);
+			while(false); // eslint-disable-line no-constant-condition
```

Comment:
![](https://media.giphy.com/media/fpXxIjftmkk9y/giphy.gif)

Reply:
I don't understand this line too 😅. @timotheeg what's this line for?